controllers/routemonitor/routemonitor_controller.go

```diff
@@ -0,0 +1,83 @@
+/*
+
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package routemonitor
+
+import (
+	"context"
+
+	"github.com/go-logr/logr"
```

Comment:
i personally prefer logrus but no hard feelings

Reply:
I didn't decide that. that is the logger of operator-sdk